module/documents/item.mjs

```diff
@@ -792,14 +793,22 @@ export default class Item5e extends SystemDocumentMixin(Item) {
       let dialogConfig = dialog;
       let messageConfig = message;
       let activity = activities[0];
-      if ( (activities.length > 1) && !event?.shiftKey ) activity = await ActivityChoiceDialog.create(this);
+      const skipDialog = areKeysPressed(event, "skipDialogNormal")
+        || areKeysPressed(event, "skipDialogAdvantage")
+        || areKeysPressed(event, "skipDialogDisadvantage");
```

Comment:
Same comment as above: Just use SHIFT for now, and keep the other modifiers in reserve.